packages/@uppy/dashboard/src/components/FileList.js

```diff
@@ -32,22 +51,34 @@ module.exports = (props) => {
     cancelUpload: props.cancelUpload,
     toggleFileCard: props.toggleFileCard,
     removeFile: props.removeFile,
-    handleRequestThumbnail: props.handleRequestThumbnail
+    handleRequestThumbnail: props.handleRequestThumbnail,
+    handleCancelThumbnail: props.handleCancelThumbnail
   }
 
-  function renderItem (fileID) {
+  const rows = chunks(Object.keys(props.files), props.itemsPerRow)
+
+  function renderRow (row) {
     return (
-      <FileItem
-        key={fileID}
-        {...fileProps}
-        file={props.files[fileID]}
-      />
+      // Use the first file ID as the key—this should not change across rerenders
+      <div role="presentation" key={row[0]}>
```

Comment:
Can we make the innermost div in VirtualList `ul`, and leave ListItem as the `li`?